EnglishFilter/sections/antiadblock.txt

```diff
@@ -504,9 +504,9 @@ iptvbin.com$$script[tag-content="a.id=""arlinablock"""][max-length="1200"]
 !
 ! ignielAdBlock
 ! For some cases, try this scriptlet `#%#//scriptlet("abort-current-inline-script", "document.createElement", "ignielAdBlock")` or JS rule `#%#AG_abortInlineScript(/ignielAdBlock/, 'document.createElement');`
-miuiku.com,tatangga.com,flash-firmware.blogspot.com,ghostsnet.com,rezkozpatch.xyz,ashort1a.xyz,orirom.com,boycracked.com,rumahit.id#%#//scriptlet("abort-current-inline-script", "document.createElement", "ignielAdBlock")
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=miuiku.com|tatangga.com|flash-firmware.blogspot.com|ghostsnet.com|rezkozpatch.xyz|rumahit.id|sekilastekno.com|romfirmware.com|orirom.com|boycracked.com|gsmfirmware.net|animekuro.net|ashort1a.xyz
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=miuiku.com|tatangga.com|flash-firmware.blogspot.com|ghostsnet.com|rezkozpatch.xyz|rumahit.id|sekilastekno.com|romfirmware.com|orirom.com|boycracked.com|gsmfirmware.net|animekuro.net|ashort1a.xyz
+seg-ashort1a-ma.tk,miuiku.com,tatangga.com,flash-firmware.blogspot.com,ghostsnet.com,rezkozpatch.xyz,ashort1a.xyz,orirom.com,boycracked.com,rumahit.id#%#//scriptlet("abort-current-inline-script", "document.createElement", "ignielAdBlock")
```

Comment:
Rules (redirect/exception) are enough here. This scriptlet is needed, if those rules aren't working.

Reply:
So What should I do? Remove this from Igniel Scriptlet?

Reply:
Yes, only this.

Reply:
What do you mean? Please be clear. Should I keep the domain in Pageadgooglesyndication or remove?

Reply:
I mean you should only remove a domain from the scriptlet.

Reply:
Fixed.